src/bundle/Resources/public/js/scripts/services/notification.service.js

```diff
@@ -0,0 +1,93 @@
+(function(global, doc) {
+    const eZ = (global.eZ = global.eZ || {});
+
+    const NOTIFICATION_INFO_LABEL = 'info';
+    const NOTIFICATION_SUCCESS_LABEL = 'success';
+    const NOTIFICATION_WARNING_LABEL = 'warning';
+    const NOTIFICATION_DANGER_LABEL = 'danger';
+
+    /**
+     * Dispatches notification event
+     *
+     * @function showNotification
+     * @param {Object} detail
+     */
+    const showNotification = (detail) => {
+        const event = new CustomEvent('ez-notify', {
+            detail,
+        });
+
+        document.body.dispatchEvent(event);
+    };
+
+    /**
+     * Dispatches info notification event
+     *
+     * @function showInfoNotification
+     * @param {String} message
+     */
+    const showInfoNotification = (message) => {
+        showNotification({
+            message,
+            label: NOTIFICATION_INFO_LABEL,
+        });
+    };
+
+    /**
+     * Dispatches success notification event
+     *
+     * @function showSuccessNotification
+     * @param {String} message
+     */
+    const showSuccessNotification = (message) => {
+        showNotification({
+            message,
+            label: NOTIFICATION_SUCCESS_LABEL,
+        });
+    };
+
+    /**
+     * Dispatches warning notification event
+     *
+     * @function showWarningNotification
+     * @param {String} message
+     */
+    const showWarningNotification = (message) => {
+        showNotification({
+            message,
+            label: NOTIFICATION_WARNING_LABEL,
+        });
+    };
+
+    /**
+     * Dispatches danger notification event
+     *
+     * @function showDangerNotification
+     * @param {String} message
+     */
+    const showDangerNotification = (message) => {
+        showNotification({
+            message,
+            label: NOTIFICATION_DANGER_LABEL,
+        });
+    };
+
+    /**
+     * Dispatches danger notification event
+     *
+     * @function showErrorNotification
+     * @param {Error} error
+     */
+    const showErrorNotification = (error) => {
+        showDangerNotification(error.message);
+    };
+
+    eZ.services = eZ.services || {};
```

Comment:
I don't know if `services` is an accurate name. Maybe `helpers`?

Reply:
I was thinking about this and I actually searched for an answer on the Internet and I encountered this thread:
https://softwareengineering.stackexchange.com/questions/132067/difference-between-a-service-class-and-a-helper-class

The top answer says: 
> A Service class/interface provides a way of a client to interact with some functionality in the application.
and for me this code, which is sending notifications does that, "provides a way of a client to interact with some functionality in the application".
If you have better definition etc. I would gladly see it because I really want to name it correctly. :)

A helper for me would be e.g. a set of functions generating `CustomEvent`s but not dispatching it to the DOM.

Reply:
IMO, a service is meant to manage data.

Reply:
@sunpietro , can you provide some reference?